pyproject.toml

```diff
@@ -21,7 +21,7 @@ pydantic = "^2.7.3"
 openai = "^1.33.0"
 posthog = "^3.5.0"
 groq = "^0.9.0"
-
+together = "^1.2.1"
```

Comment:
Lets put together in the optional dependency. Feel free to do it in a follow up PR.